examples/webpack5/start.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+set -eux
+
+readonly workspace=$(bazel info workspace)
+# The cd here is not necessary, but if you were to run in a
+# nested package in a large workspace like a monorepo you can 
+# use the cd to set your working directory so webpack runs correctly
+bazel run --run_under="cd ${workspace} && " //:serve
```

Comment:
Bazel always runs executables at `execRoot` (see https://docs.bazel.build/versions/master/output_directories.html)
Even in a directory with nested packages, you would need only point webpack to the right entry point within the nested directory structure. This can be accomplished using webpack's `--entry` flag or via the `entry` configuration option within your webpack config

Reply:
Alternatively you could just set `chdir` on the webpack target to get the same behavior

Reply:
So this is a weird example because I ported it form internal code where the `--run_under` shifts the directory into a nested directory. This removes the need for subprocesses like webpack to have special paths past to them. I think the problem I'm trying to solve for here is that a lot of JS tooling assumes the `cwd` is where the project is. That's not true with Bazel, and thus makes setting things up kind of fragile. I'd rather meet the tooling where it currently is, than make the tooling meet Bazel where it is.

I'm not following how the `execroot` is better for the underlying file watcher. Wouldn't watching more unnecessary files be more expensive for webpack instead of only watching the files it is concerned with?